qiskit/providers/backend.py

```diff
@@ -505,6 +505,26 @@ def qubit_properties(
             return self.target.qubit_properties[qubit]
         return [self.target.qubit_properties[q] for q in qubit]
 
+    def get_qubit_channels(self, qubit: int):
```

Comment:
Can you implement this method as a helper function in the `pulse.builder` or `pulse.utils`? I want to backport this PR if possible, but this introduces API change and prevents us from the patch release with this. I also don't think there is demand for this method (i.e. V1 and V2 backend interface doesn't need to be fully compatible).

Reply:
Fixed at https://github.com/Qiskit/qiskit-terra/pull/10150/commits/a21cc84d657c3cc5b9d1d6a8b0d22c733c8bae32.